pom.xml

```diff
@@ -201,14 +201,14 @@
     </build>
     <profiles>
         <profile>
-            <id>spark33</id>
+            <id>spark34</id>
```

Comment:
@shivsood Could you confirm that 3.4 spark DBR environment could not use 3.3 connector?

Reply:
1.3.0-BETA release ( Spark 3.3) throws an exception for get schema type with DBR Spark 3.4.0.